liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppMojoSupport.java

```diff
@@ -85,7 +86,14 @@ protected void installApp(Artifact artifact) throws Exception {
     
     // install project artifact using loose application configuration file 
     protected void installLooseConfigApp() throws Exception {
-        File destDir = new File(serverDirectory, appsDirectory);
+    
+        setAppsDirectory(); 
+    
+        if (getAppsDirectory().equalsIgnoreCase("apps") && !isApplicationConfigured()) {
+            addAppConfiguration(project.getBuild().getFinalName());
```

Comment:
What happen if stripVersion = false and the application filename contains the version string?